examples/spec/integration/serial_hello_world_workflow_spec.rb

```diff
@@ -4,13 +4,11 @@
   it 'completes' do
     workflow_id, run_id = run_workflow(described_class, 'Alice', 'Bob', 'John')
 
-    result = fetch_history(
-      workflow_id,
-      run_id,
-      wait_for_new_event: true,
-      event_type: :close,
+    result = Temporal.await_workflow_result(
```

Comment:
👍 , I suppose for *our* specs it's not that useful because we're not really testing product behavior.
It is a good example for integrating teams to copy though.